server/routes/recipes.js

```diff
@@ -0,0 +1,107 @@
+const router = require('express').Router();
+const mongoose = require('mongoose');
+
+let Recipe = require('../models/recipe.js')
+
+router.route("/").get((req, res) => {
+  return res.send("test");
+});
+
+//Create Recipe
+router.route("/").post((req, res) => {
+  const _id = new mongoose.Types.ObjectId();
+  const name = req.body.name;
+  const ingredients = req.body.ingredients;
+  const description = req.body.description;
+  const price = req.body.price;
+  const chef = req.body.chef;
+  const servingSize = req.body.servingSize;
+  const requiredStuff = req.body.requiredStuff;
+  const picture = req.body.picture;
```

Comment:
You can do a nice and easy [object destructuring](https://wesbos.com/destructuring-objects)!
`const { name, ingredients, description, price, chef, servingSize, picture } = req.body;`